collections/climate-change-initiative.yaml

```diff
@@ -0,0 +1,83 @@
+          Title: Description
+      Rows:
+        - Name: Aerosol
+          Description: Set of global aerosol ECV products from different sensors
+        - Name: Cloud
+          Description: Long-term coherent cloud property data sets
+        - Name: Greenhouse Gases
+          Description: Carbon Dioxide and Methane products
+        - Name: Ozone
+          Description: Ozone products from ESA and Third Party Missions
+        - Name: Ocean Colour
+          Description: Chlorophyll, Inherent Optical Properties, and Water-leaving Reflecatnce products
+        - Name: Sea Ice
+          Description: Sea ice Concentration and Sea ice Thickness products
+        - Name: Sea Level
+          Description: Sea Level ECV products from different instruments
+        - Name: Sea State 
+          Description: Wave height products integrating different sensors
+        - Name: Sea Surface Salinity
+          Description: Sea Surface Salinity products 
+        - Name: Sea Surface Temperature
+          Description: Sea Surface Temperature products
+        - Name: Water Vapour
+          Description: Products on Water Vapour content in the Atmosphere     
+        - Name: Antarctic Ice Sheet
+          Description: Location, Mass Balance, and Velocities of the Antarctic Ice Sheet
+        - Name: Above-Ground Biomass 
+          Description: Global Datasets for forest Above-Ground Biomass 
+        - Name: Fire 
+          Description: Different Burned Area products
+        - Name: Glaciers
+          Description: Inventory of Glacier Outlines  
+        - Name: Greenland Ice Sheet
+          Description: Different products characterizing the dynamics of Greenland Ice Sheet
+        - Name: Lakes 
+          Description: Different variables characterizing Lakes on a global scale
+        - Name: Land Cover
+          Description: Global classification of Land Cover and a Water Bodies Map
+        - Name: Land Surface Temperature
+          Description: Land Surface Temperature products integrating different sensors 
+        - Name: Permafrost  
+          Description: Active Layer Thickness, Ground Temperature, and Extent products for the Northern Hemisphere
+        - Name: Snow
+          Description: Long time-series of daily Snow Water Equivalent and Snow Cover Fraction
+        - Name: Soil Moisture
+          Description: Soil Moisture products from different sensors   
+          
+Contact: "[CCI Open Data Portal Helpdesk](https://climate.esa.int/en/helpdesk/)"
+Provider: "[ESA Climate Office](https://climate.esa.int/en/esa-climate/esa-climate-office/)"
+ManagedBy: "[Brockmann Consult](https://www.brockmann-consult.de/)"
+Tags:
+    - Climate
+    - CCI
+    - open data
+    - xcube
+    - raster
+    - essential climate variables
+License: |
+  ESA CCI Data Policy: free and open access ([details](https://climate.esa.int/sites/default/files/CCI_Data_Policy_v1.1.pdf)).
+Resources:
+ - Group: xcube Resources
+ - StoreID: cciodp

```

Comment:
Is this single reference to the `StoreID` sufficient to access the data? For example. in [this ocean dataset](https://github.com/eurodatacube/public-collections/blob/alicja-adding-ESDC-description/collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml) there is more resource information listed. You can follow the structure as shown below:

![image](https://user-images.githubusercontent.com/60575896/204850022-4710b4c9-4f0c-446a-bb18-c4315b9947ae.png)

Reply:
The problem here is that the store provides access to 413 datasets, each with several variables. In total, probably more than 4000 variables. It does not seem too practical to list them all on a single variable level.